CSharp/BatchExplorer/Helpers/EnumerableExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.BatchExplorer.Helpers
+{
+    internal static class EnumerableExtensions
+    {
+        internal static IEnumerable<T> SelectMany<T>(this IEnumerable source, Func<object, int, IEnumerable<T>> selector)
```

Comment:
If I had an IEnumerable<TSource> then I wouldn't need to write a custom extension.